includes/forum-reactions.php

```diff
@@ -169,36 +174,43 @@ public function reaction_change($post_id, $user_id, $reaction) {
 
         // Add reaction when there is none.
         if ($reaction_check === false) {
-            $this->add_reaction($post_id, $user_id, $reaction);
+            $this->add_reaction($post_id, $user_id, $reaction, $author);
         }
 
         // Remove reaction when it is already set.
         if ($reaction_check === $reaction) {
-            $this->remove_reaction($post_id, $user_id, $reaction);
+            $this->remove_reaction($post_id, $user_id, $reaction, $author);
```

Comment:
I dont think we need the `$author` variable to remove the reaction because the combination of `$post_id` and `$user_id` already ensures that it is a unique combination.

Reply:
You are absolutely right.
I will revert this change.